### ISSUE TITLE
Creating tweet with video fails due to the program trying to post the status too soon

### ISSUE THREAD

**Issue by karoliina** [liu044100/SocialVideoHelper#10]
When trying to upload a video to Twitter, I noticed that I was getting a response with the status code 400 and the error

```
{
    code = 324;
    message = "Not valid video";
}
```

in `tweetVideoStage4`. The problem is due to the media not yet being available on Twitter's servers when `tweetVideoStage4` tries to post a status using the media ID of the video that was just uploaded. The stage 3 response alludes to this with its message

```
"processing_info": {
    "state": "pending",
    "check_after_secs": 5
}
```

I solved this by adding the line `[NSThread sleepForTimeInterval:5.0f];` before the final HTTP request is sent in `tweetVideoStage4`.

**Comment by liu044100** [comment]
@karoliina It seems the https://github.com/liu044100/SocialVideoHelper/pull/9 will solve this issue. I will merge it as soon as possible.

**Comment by liu044100** [comment]
@karoliina I merged #9, you can try it.

**Comment by karoliina** [comment]
That PR fixed the issue, thanks!
